TestAccCloudflareCustomHostnameFallbackOrigin is failing

`TestAccCloudflareCustomHostnameFallbackOrigin` is failing with the following exception due to not supporting API tokens

```
=== RUN   TestAccCloudflareCustomHostnameFallbackOrigin
    testing.go:705: Step 0 error: errors during apply:
        
        Error: failed to create custom hostname fallback origin: error from makeRequest: HTTP status 501: content "{\"success\":false,\"errors\":[{\"code\":10000,\"message\":\"API Tokens are not supported by this API for now\"}]}\n"
        
          on /tmp/tf-test447682186/main.tf line 2:
          (source code not available)
        
        
--- FAIL: TestAccCloudflareCustomHostnameFallbackOrigin (7.59s)
=== RUN   TestAccCloudflareCustomHostnameFallbackOriginUpdate
    testing.go:705: Step 0 error: errors during apply:
        
        Error: failed to create custom hostname fallback origin: error from makeRequest: HTTP status 501: content "{\"success\":false,\"errors\":[{\"code\":10000,\"message\":\"API Tokens are not supported by this API for now\"}]}\n"
        
          on /tmp/tf-test623878732/main.tf line 2:
          (source code not available)
```

Link to failure: https://github.com/cloudflare/terraform-provider-cloudflare/runs/1295364502?check_suite_focus=true#step:6:212

--- a/cloudflare/resource_cloudflare_access_identity_provider_test.go
+++ b/cloudflare/resource_cloudflare_access_identity_provider_test.go
@@ -46,6 +46,16 @@ func testSweepCloudflareAccessIdentityProviders(r string) error {
 }
 
 func TestAccCloudflareAccessIdentityProviderOneTimePin(t *testing.T) {
+	// Temporarily unset CLOUDFLARE_API_TOKEN if it is set as the OTP Access
+	// endpoint does not yet support the API tokens for updates and it results in
+	// state error messages.
+	if os.Getenv("CLOUDFLARE_API_TOKEN") != "" {
+		defer func(apiToken string) {
+			os.Setenv("CLOUDFLARE_API_TOKEN", apiToken)
+		}(os.Getenv("CLOUDFLARE_API_TOKEN"))
+		os.Setenv("CLOUDFLARE_API_TOKEN", "")
+	}
+
 	t.Parallel()
 	rnd := generateRandomResourceName()
 	resourceName := "cloudflare_access_identity_provider." + rnd

--- a/cloudflare/resource_cloudflare_custom_hostname_fallback_origin_test.go
+++ b/cloudflare/resource_cloudflare_custom_hostname_fallback_origin_test.go
@@ -11,6 +11,16 @@ import (
 )
 
 func TestAccCloudflareCustomHostnameFallbackOrigin(t *testing.T) {
+	// Temporarily unset CLOUDFLARE_API_TOKEN if it is set as the custom hostname
+	// fallback endpoint does not yet support the API tokens for updates and it
+	// results in state error messages.
+	if os.Getenv("CLOUDFLARE_API_TOKEN") != "" {
+		defer func(apiToken string) {
+			os.Setenv("CLOUDFLARE_API_TOKEN", apiToken)
+		}(os.Getenv("CLOUDFLARE_API_TOKEN"))
+		os.Setenv("CLOUDFLARE_API_TOKEN", "")
+	}
+
 	zoneID := os.Getenv("CLOUDFLARE_ZONE_ID")
 	domain := os.Getenv("CLOUDFLARE_DOMAIN")
 	rnd := generateRandomResourceName()
@@ -50,6 +60,16 @@ resource "cloudflare_record" "%[2]s" {
 }
 
 func TestAccCloudflareCustomHostnameFallbackOriginUpdate(t *testing.T) {
+	// Temporarily unset CLOUDFLARE_API_TOKEN if it is set as the custom hostname
+	// fallback endpoint does not yet support the API tokens for updates and it
+	// results in state error messages.
+	if os.Getenv("CLOUDFLARE_API_TOKEN") != "" {
+		defer func(apiToken string) {
+			os.Setenv("CLOUDFLARE_API_TOKEN", apiToken)
+		}(os.Getenv("CLOUDFLARE_API_TOKEN"))
+		os.Setenv("CLOUDFLARE_API_TOKEN", "")
+	}
+
 	zoneID := os.Getenv("CLOUDFLARE_ZONE_ID")
 	domain := os.Getenv("CLOUDFLARE_DOMAIN")
 	rnd := generateRandomResourceName()

